### DISCUSSION TITLE
when call url audio

### DISCUSSION THREAD

**Discussion by fe2o0o** [katspaugh/wavesurfer.js#3887] [Q&A]
when i use url for audio it have issue 
WaveSurfer Error: EncodingError: Failed to execute 'decodeAudioData' on 'BaseAudioContext': Unable to decode audio data

![wavesuffer](https://github.com/user-attachments/assets/092d6d9b-a302-45c4-8c44-d1867798239c)

please i need help

**Comment by katspaugh** [comment]
Sounds like the file you're loading cannot be decoded.

**Reply by fe2o0o** [reply]
why but when i try loading from local it's loaded successfully but i use cloud url like cloudinary or amazons3 it loadded and sound play but waves not show

**Comment by fe2o0o** [comment]
when i try convert it to buffer and pass buffer array it works but not all audios

**Comment by fe2o0o** [comment]
i using it with angular 14

**Comment by waelantar** [comment]
same problem here with angular 18 @katspaugh and @fe2o0o  is there any solution
